### PR TITLE
Updated French translation

### PR DESCRIPTION
**Pull request by rene-coty**
I updated the fr.po file

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -2389,53 +2338,51 @@ msgstr "Nettoyage"
...
 #: share/gpodder/ui/gtk/gpodderpreferences.ui.h:32
 msgid "Only sync unplayed episodes"
 msgstr "Ne synchroniser que les épisodes non lus"
 
 #: share/gpodder/ui/gtk/gpodderpreferences.ui.h:33
-#, fuzzy
-#| msgid "Remove episodes deleted on device from gPodder"
 msgid "Remove episodes deleted in gPodder from device"
-msgstr "Retirer de gPodder les épisodes supprimés sur le périphérique"
+msgstr "Retirer les épisodes supprimés de gPodder du périphérique"
```
Comment (elelay) [inline po/fr.po:2369]: Retirer du périphérique les épisodes supprimés dans gPodder

**Review by elelay — CHANGES_REQUESTED** [review]
Thank you for the translation.
Looks good to me except one sentence that I would rephrase (see comment).

Also I'm not sure why you removed the space following colon, which I would consider necessary in French. Please explain :-)

**Comment by rene-coty** [comment]
> Thank you for the translation. Looks good to me except one sentence that I would rephrase (see comment).

Okay I'll change it :+1:

**Comment by rene-coty** [comment]
Normally there is no need of a space following colon in a French translation (if there is no text after it) ; it is only required to have an unbreakable space before a colon.

**Comment by elelay** [comment]
Merged, thanks
